service/src/chain_spec.rs

```diff
@@ -126,6 +127,10 @@ fn staging_testnet_config_genesis() -> GenesisConfig {
 		curated_grandpa: Some(CuratedGrandpaConfig {
 			shuffle_period: 1024,
 		}),
+		im_online: Some(ImOnlineConfig{
```

Comment:
```suggestion
		im_online: Some(ImOnlineConfig {
```

what's with the missing spaces habit?

Reply:
Thanks for catching it. Sometimes, in my wildest dreams, I dream of a linter in the CI.